lib/DxilDia/DxcPixLiveVariables.cpp

```diff
@@ -232,7 +264,8 @@ HRESULT dxil_debug_info::LiveVariables::GetLiveVariablesAtInstruction(
         LiveVars.emplace_back(VarAndInfo.second.get());
       }
     }
-    S = S->getScope().resolve(EmptyMap);
+    S.second = nullptr;
```

Comment:
Is there user of S.second?

Reply:
S.second is only there to be part of the key for the m_LiveVarsDbgDeclare map. Seems like the "inlined at" function is always the top-level function even when the inlined is more than one function deep, so it's OK to discard "S.second": it has done its job (unique-ifying the subroutine AND where it was inlined).

Reply:
I see. It would be nice to have some comment for this.